src/Sylius/Bundle/AdminBundle/Resources/views/Order/Show/Summary/_totals.html.twig

```diff
@@ -52,18 +60,22 @@
 </tr>
 <tr>
     <td colspan="4" id="promotion-discounts">
-        <strong>{{ 'sylius.ui.discount'|trans }}</strong>:
         {% set orderPromotions = sylius_aggregate_adjustments(order.getAdjustmentsRecursively(orderPromotionAdjustment)) %}
         {% if not orderPromotions is empty %}
-            <ul class="ui relaxed list">
+            <div class="ui relaxed divided list">
                 {% for label, amount in orderPromotions %}
-                    <li class="item">
-                        {{ label }} {% if not amount == 0 %}-{% endif %}{{ (-1 * amount)|sylius_price(order.currencyCode, order.exchangeRate) }}
-                    </li>
+                    <div class="item">
+                        <div class="content">
+                            <span class="header">{{ label }}</span>
+                            <div class="description">
+                                {% if not amount == 0 %}-{% endif %}{{ (-1 * amount)|sylius_price(order.currencyCode, order.exchangeRate) }}
```

Comment:
Here and everywhere else:

`{% if not amount == 0 %}-{% endif %}{{ (-1 * amount)|sylius_price(order.currencyCode, order.exchangeRate) }}`

can be replaced with just:

`{{ amount|sylius_price(order.currencyCode, order.exchangeRate) }}`

Seems like it was a workaround for some currently fixed bug.